<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget {
	background-image: url(:/imgBackground/background.jpg);
}
#Widget * { 
border-image:url(); 
}</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QLabel" name="lblSys">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1024</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>智能体重管理系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>187</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>247</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密  码:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTourTest">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>405</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/imgIcon/ico.png);
	border-style:hidden
}
QPushButton:hover
{
	margin:+5
}
QPushButton:pressed
{
	margin:+10
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblToursit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>565</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>游客测重</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>404</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/imgIcon/poke-ball.png);
	border-style:hidden
}

QPushButton:hover
{
	margin:+5
}
QPushButton:pressed
{
	margin:+10
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>404</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/imgIcon/ico2.png);
	font: 25 20pt &quot;Roboto&quot;;
	color: rgb(224, 77, 77);
	border-style:hidden
}

QPushButton:hover
{
	margin:+5
}
QPushButton:pressed
{
	margin:+10
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegister">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>565</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblToursit_2">
   <property name="geometry">
    <rect>
     <x>804</x>
     <y>560</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
